src/tactic/core.lean

```diff
@@ -1033,6 +1019,51 @@ meta def mk_meta_pis : expr → tactic (list expr × expr)
   return ((p :: ps), r)
 | e := return ([], e)
 
+end tactic
+
+namespace lean.parser
```

Comment:
I've moved around the `lean.parser` section of this file to later on after the general tactic functions, since the code here might want to use them and parser functions cannot themselves be called from tactic-land (and I needed one such function).